lib/sportradar/api/nfl/game_statistic.rb

```diff
@@ -0,0 +1,75 @@
+module Sportradar
+  module Api
+    class Nfl::GameStatistic < Data
+      attr_accessor :response, :id, :efficiency, :first_downs, :interceptions, :touchdowns, :name, :market, :alias, :reference, :possession_time, :avg_gain, :safeties, :turnovers, :play_count, :rush_plays, :total_yards, :lost_fumbles, :penalty_yards, :return_yards
+
+      # attr_writer :rushing, :receiving, :punts, :punt_returns, :penalties, :passing, :misc_returns, :kickoffs, :kick_returns, :int_returns, :fumbles, :field_goals, :extra_points, :defense # probably not necessary, but leaving here in case we want it later
+
+      def initialize(data)
+        @response        = data
+        @id              = data["id"]
+        @name            = data["name"]
+        @market          = data["market"]
+        @alias           = data["alias"]
+        @reference       = data["reference"]
+        @efficiency      = data["efficiency"]
+        @first_downs     = data["first_downs"]
+        @interceptions   = data["interceptions"]
+        @touchdowns      = data["touchdowns"]
+        @possession_time = data["possession_time"]
+        @avg_gain        = data["avg_gain"]
+        @safeties        = data["safeties"]
+        @turnovers       = data["turnovers"]
+        @play_count      = data["play_count"]
+        @rush_plays      = data["rush_plays"]
+        @total_yards     = data["total_yards"]
+        @lost_fumbles    = data["lost_fumbles"]
+        @penalty_yards   = data["penalty_yards"]
+        @return_yards    = data["return_yards"]
+      end
+      def rushing
```

Comment:
Is the idea behind all these methods just to not bother setting the classes until they are invoked, rather than the initializer?  Makes sense if they are "optionally" used